WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserAdapter.java

```diff
@@ -65,6 +68,7 @@ public View newView(Context context, Cursor cursor, ViewGroup parent) {
 
         ThemeViewHolder themeViewHolder = new ThemeViewHolder(view);
         view.setTag(themeViewHolder);
+        mViewWidth = parent.getWidth();
```

Comment:
> I'll change it so the width is passed in from the Fragment where I can divide the width by the number of columns

Yep I think it's the best solution, the other solution is to get the grid view size, but you have to wait until it's "measured" (you can't call `getWidth()` in the adapter).